src/test/java/com/powsybl/sld/server/SingleLineDiagramTest.java

```diff
@@ -349,6 +349,48 @@ void testNetworkAreaDiagram() throws Exception {
                 .andExpect(status().isNotFound());
     }
 
+    @Test
+    void testNetworkAreaDiagramFromConfig() throws Exception {
+        UUID testNetworkId = UUID.fromString("7928181c-7977-4592-ba19-88027e4254e4");
```

Comment:
Do not use constant string uuid but use `UUID.random`

Reply:
done